src/System.IO.Pipes/src/System/IO/Pipes/NamedPipeServerStream.cs

```diff
@@ -108,6 +108,9 @@ public NamedPipeServerStream(String pipeName, PipeDirection direction, int maxNu
             {
                 throw new ArgumentOutOfRangeException("maxNumberOfServerInstances", SR.ArgumentOutOfRange_MaxNumServerInstances);
             }
+            // inheritability will always be None since this private constructor is only called from other constructors from which
```

Comment:
Interesting.  @ericstj, was the removal of this ctor from the public surface area for .NET Core by design?

If so, we should remove the associated code from the implementation.

If not, we should look at putting the ctor back into the contract.